### DISCUSSION TITLE
Errors in Log - Proof Found but no Reward

### DISCUSSION THREAD

**Discussion by DendyDarzartar** [Chia-Network/chia-blockchain#7927] [Support]
I had found two proofs but did not get a payout. After doing some researching to see what might cause this I was led to look for null values in the .yaml and this stood out. Should the [] be filled in with my public pool key?

```
farmer:
  # The farmer server (if run) will run on this port
  port: 8447
  # The farmer will attempt to connect to this full node and harvester
  full_node_peer:
    host: *self_hostname
    port: 8444
  harvester_peer:
    host: *self_hostname
    port: 8448

  pool_public_keys: []

  # Replace this with a real puzzle hash
  # xch_target_address: txch102gkhhzs60grx7cfnpng5n6rjecr89r86l5s8xux2za8k820cxsq64ssdg
```

These are the errors that I am concerned with

```
2021-08-01T14:28:29.125 wallet chia.wallet.wallet_node    : WARNING  No keys present. Create keys with the UI, or with the 'chia keys' program.
2021-08-01T14:33:04.706 wallet asyncio                    : ERROR    Task exception was never retrieved
future: <Task finished name='Task-118' coro=<WalletPeers.ensure_is_closed() done, defined at /home/dendy/chia-blockchain/chia/server/node_discovery.py:701> exception=AttributeError("'WalletPeers' object has no attribute 'connection'")>
Traceback (most recent call last):
  File "/home/dendy/chia-blockchain/chia/server/node_discovery.py", line 704, in ensure_is_closed
    await self._close_common()
  File "/home/dendy/chia-blockchain/chia/server/node_discovery.py", line 118, in _close_common
    await self.connection.close()
AttributeError: 'WalletPeers' object has no attribute 'connection'
2021-08-01T14:33:11.372 harvester chia.harvester.harvester: WARNING  Not farming any plots on this harvester. Check your configuration.
2021-08-01T14:49:39.956 harvester chia.harvester.harvester: WARNING  Not farming any plots on this harvester. Check your configuration.
```

**Comment by mariano54** [comment]
Are you using a remote harvester? If so, that's fine

If not, then there is a problem. You would need to delete the .chia directory and resync.

Finding proofs but not getting rewards happens, although not very common.

**Reply by DendyDarzartar** [reply]
No, I am not running a remote harvester. Just plotting and farming on one device. 
I actually just reimaged my linux system and reinstalled chia yesterday, it finished syncing roughly 12 hours ago. 

Is it possible there could be some other cause of something else that would give this error? 
I would assume an issue with a sync that just started and finished would be unlikely but if that is the only way forward then I suppose I don't have a choice since I'm not seeing any rewards after finding two proofs today.
Although I wonder, if it is not farming as the log states, why do I have plots passing filter and on the rare occasion passing proof?

Thank you for taking the time to help me with this!

**Comment by DendyDarzartar** [comment]
Does anyone have any additional suggestions?

Please advise.
